osu.Game/Overlays/WikiOverlay.cs

```diff
@@ -145,12 +147,13 @@ private void onPathChanged(ValueChangedEvent<string> e)
         private void onLangChanged(ValueChangedEvent<Language> e)
         {
             // Path unmodified, just reload the page with new language value.
-            loadPage(path.Value, e.NewValue);
+            loadPage(path.Value == "error" ? lastValidPath : path.Value, e.NewValue);
```

Comment:
So changing language implicitly reloads the error page? Why would it do that? So weird.